src/deb/helios-agent/helios-agent-wrapper

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
I think it would be good to add some comments here about why we had to add this wrapper script (since the `EnvironmentFile` etc directives in the unit file don't support running arbitrary shell scripts)